build.gradle.kts

```diff
@@ -8,10 +8,12 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
```

Comment:
Please add to readme that you need to run server from it's main in the file and then run plugin